ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/create_table.md

```diff
@@ -143,9 +143,9 @@ Here, key is the name of the parameter and value is its value.
 
 For a list of valid parameter names and values, see the [{{ backend_name }} table description]({{ concept_table }}).
 
-For example, this code will create a table with enabled automatic partitioning by partition size and the preferred size of each partition is 512 MB:
+### Examples of Creating a String Table with Additional Parameters
```

Comment:
```suggestion
### Examples of creating a table with additional parameters
```
This is a common way to write headers, but we're not currently using it in other places.